TFGCasper.thy

```diff
@@ -0,0 +1,75 @@
+theory TFGCasper
+
+imports Main HOL.Real CBCCasper ConsensusSafety ExampleProtocols
```

Comment:
We don't need `ConsensusSafety` ?

Reply:
`consensus_value_property` type_synonym is defined in there.

Reply:
hmm... OK!